File: 백준/Bronze/2753. 윤년/윤년.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.time.LocalDate;
import java.time.Month;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int year = Integer.parseInt(br.readLine());
        LocalDate date = LocalDate.of(year, Month.FEBRUARY, 1);
        System.out.println(date.isLeapYear() ? 1 : 0);
    }
}

File: 백준/Bronze/2753. 윤년/윤년.js
const fs = require('fs');

const year = fs.readFileSync('./dev/stdin')
    .toString();

function isLeapYear(year) {
    const now = new Date(year, 1, 29);
    return (now.getMonth() == 1) ? 1 : 0;
}

console.log(isLeapYear(Number(year)));